app/views/download.scala.html

```diff
@@ -107,7 +108,15 @@ <h3 id="older-versions">Previous releases</h3>
       <table class="releases">
       @releases.previous.groupBy(_.version.slice(0, 3)).toSeq.sortBy(_._1).reverse.map { group =>
         <tbody class="version">
-          <tr><th colspan="3"><h4>@group._1</h4></th></tr>
+          <tr>
+            <th colspan="3">
+              <h4 class="previousVersionHeading">@group._1</h4>
+              @if(group._1 > "2.2"){
```

Comment:
I don't think we need this check. The Installing page exists and is useful in older versions too.

Reply:
yes since there's documentation for each version we can point to them. But 1.x.x modules uses "install" while 2.x.x uses "Installing" as pages. Maybe I can add a separate route in controllers/documentation/Router resolving them to the correct places.